plugins/catalog-backend-module-ldap/src/processors/LdapOrgReaderProcessor.ts

```diff
@@ -43,17 +43,26 @@ export class LdapOrgReaderProcessor implements CatalogProcessor {
   private readonly userTransformer?: UserTransformer;
 
   static fromConfig(
-    config: Config,
+    configRoot: Config,
     options: {
       logger: Logger;
       groupTransformer?: GroupTransformer;
       userTransformer?: UserTransformer;
     },
   ) {
-    const c = config.getOptionalConfig('catalog.processors.ldapOrg');
+    // TODO(freben): Deprecate the old catalog.processors.ldapOrg config
+    const config =
+      configRoot.getOptionalConfig('ldap') ||
+      configRoot.getOptionalConfig('catalog.processors.ldapOrg');
+    if (!config) {
+      throw new TypeError(
```

Comment:
technically this is a change of behavior but it used to not work at all without it anyway...